packages/manager/modules/hub/src/components/order-tracking/index.scss

```diff
@@ -18,4 +18,14 @@
     font-size: $hub-subtitle-size;
     font-weight: $hub-subtitle-weight;
   }
+
+  a.oui-status {
+    padding: 0.5rem;
+    text-decoration: none;
+    background-color: #bef1ff; // P-100
```

Comment:
What do the comments mean ? They don't look like they are linked to the background colors, are they ?

Reply:
it's the ODS color codes